### PR TITLE
Arrays - Sum Of Diagonals

### PR DESCRIPTION
**Pull request by Abduliitr**
## Checklist
- Solution is tested on sphere engine <br/>
    `Note:` Please copy only the solution part. Do not copy/paste the template code. That way, any issues with template code can be caught early on.
    - [x] Author
    - [x] Reviewer
- PR title is correct - [PR title conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#title-for-a-pull-request)
    - [x] Author
    - [x] Reviewer
- Folder/File naming is correct - [Naming conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#naming-of-files-and-folders)
    - [x] Author
    - [x] Reviewer
- Coding conventions are followed - [Coding conventions](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#coding-conventions)
    - [x] Author
    - [x] Reviewer
- Test cases (action item for **reviewer**) <br/>
    - To be ensured by checking the problem on sphere-engine
    - If any test cases are missing, please add them - [how to add new test cases](https://github.com/DSA-solutions-the10xacademy/JavaScript-DSA-Solutions#adding-new-test-cases)
    - [x] There are at least 2 test cases
    - [x] There is at least 1 test case that is different from the sample test cases

### CODE REVIEW

**Review by LVSsandeepVarma — APPROVED** [review]
code looks good , but while working on video, @prathyusha10x make sure that whoever records this video   explain what  `+=`  is , as many students might not know about it

File: solutions/arrays/sum-of-diagonals.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+let sumSecondDiagonal = 0;
+
+for(let i = 0; i < matrix.length; i++){
+    let j = i;
+    sumFirstDiagonal += matrix[i][j];
+}
+
+for(let i = 0; i < matrix.length; i++){
+    let j = matrix.length - 1 - i;
+    sumSecondDiagonal += matrix[i][j];
```
Comment (sailesh-ramayanam) [inline solutions/arrays/sum-of-diagonals.js:41]: @Abduliitr 
1. One for loop is sufficient for both diagonals 
2. No need of `j` - normally I would have preferred the use of `j`. In this case I want the students to see the computation of diagonal indices.

**Review by sailesh-ramayanam — CHANGES_REQUESTED** [review]

**Comment by DevratAgarwal** [comment]
@LVSsandeepVarma @sailesh-ramayanam changes are addressed

**Comment by prathyusha10x** [comment]
@sailesh-ramayanam @LVSsandeepVarma Please review the Updated codes and approve the PR by 21 march 2022

**Comment by LVSsandeepVarma** [comment]
> @LVSsandeepVarma @sailesh-ramayanam changes are addressed

@DevratAgarwal  I did'nt see any changes in the code , did you commit your changes?

**Comment by DevratAgarwal** [comment]
@LVSsandeepVarma sorry it was not my question.

**Review by sailesh-ramayanam — APPROVED** [review]

**Pull request merged**